Cargo.toml

```diff
@@ -49,9 +51,4 @@ features = ["runtime-actix", "postgres", "chrono", "offline", "macros"]
 git = "https://github.com/launchbadge/sqlx/"
 branch = "master"
 default-features = false
-features = ["runtime-actix", "postgres", "chrono", "offline"]
-
-[dependencies.meilisearch-sdk]
-# Temp fork with some patches
-git = "https://github.com/Aeledfyr/meilisearch-rust"
-branch = "master"
+features = ["runtime-actix", "postgres", "chrono", "offline"]
```

Comment:
Adding a newline at the end of this file would be useful for git, to make potential merges/rebasing easier.